code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -10,8 +10,6 @@
 	w_class = W_CLASS_TINY
 	var/uses = 1
 	var/nofail = 0
-	var/is_bullet = 0
-	var/inuse = 0
```

Comment:
And how will you prevent it from being used in two mobs at once?

Reply:
Unless it is infinite it should disappear as soon as it is used up

Reply:
It still allows you to stack do_afters. There's no reason to remove this.

Reply:
Outdated code, if anything it will just make possible future dealing with stacked do_afters easier because this snowflaked thing doesn't exist anymore

Reply:
There's nothing wrong with having a busy var, don't remove it. You shouldn't be able to physically inject two different mobs at the same time.

Reply:
fine but I'm blaming you if I forget about it if I ever make that do_after PR